.circleci/config.yml

```diff
@@ -15,24 +15,21 @@ jobs:
           git config --global user.name LinuxbrewTestBot
           git config --global user.email testbot@linuxbrew.sh
       - run:
-        name: Set up Linuxbrew
-        working_directory: /home/linuxbrew/.linuxbrew/Homebrew
-        no_output_timeout: 5m
-        command: |
-          git pull --ff-only
+        name: Create a directory for Linux taps
+        command: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
       - run:
         name: Set up Linuxbrew/$CIRCLE_PROJECT_REPONAME
-        no_output_timeout: 5m
+        working_directory: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
         command: |
+          git clone $CIRCLE_REPOSITORY_URL $CIRCLE_PROJECT_REPONAME
```

Comment:
Note that `$CIRCLE_REPOSITORY_URL` uses ssh. I'd suggest using
```
git clone https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
```

Reply:
The second `$CIRCLE_PROJECT_REPONAME` is redundant, since it's the default.

Reply:
Hmmm, according to https://circleci.com/docs/1.0/environment-variables/, it's `https://...`

Reply:
oh, it might be 1.0-specific